server/src/fat/java/com/ibm/ws/lars/rest/PermissionTest.java

```diff
@@ -50,7 +50,7 @@
     @Rule
     public RepositoryContext adminContext;
 
-    @Rule
+    // This is not a rule, as a rule does setup/tear down which is not wanted.
```

Comment:
As it's no longer a rule, does it need to be closed?

I see that RepositoryContext is now Closeable since the HttpClient is closeable. Unfortunately, I can't find any information on whether you need to close the HttpClient or not.